app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Admin
+  class SessionsController < ApplicationController
+    def new
+    end
+    def create
+      if params[:email].present?
```

Comment:
email = if params[:email].present?
params[:email].downcase
else
params[:email]
end

Reply:
A to nie ładniej nawet 
email = params[:email].present? : email = params[:email].downcase ? params[:email]
?

Reply:
Jak się mieści w jednej linijce to ładniej tylko składnię masz tu na odwrót (? `zrób to co jeśli true` : `zrób to co jeśli false`)

Reply:
Poprawione